rpm/qtdeclarative.spec

```diff
@@ -268,6 +285,14 @@ touch .git
 
 make %{?_smp_mflags}
 
+%if 0%{?qt5_release_build}
```

Comment:
Yep I was thinking of testing it without adding the flag directly so I can make sure it's the right flag but I'll try running it through OBS now, cheers.